cobald_tests/daemon/config/test_python.py

```diff
@@ -0,0 +1,29 @@
+import tempfile
+import os
+
+from cobald.daemon.config.python import load_configuration
+
+
+def module_content(identifier):
+    return f"""\
+identifier = {identifier}
+"""
+
+
+def test_load_pyconfig():
+    with tempfile.NamedTemporaryFile(mode="w+", suffix=".py") as test_file:
+        test_file.write(module_content(0))
+        test_file.flush()
+        module = load_configuration(test_file.name)
+        assert module.identifier == 0
+
+
+def test_load_pyconfig_many():
+    modules = []
+    for ident in range(5):
+        with tempfile.NamedTemporaryFile(mode="w+", suffix=".py") as test_file:
+            test_file.write(module_content(ident))
+            test_file.flush()
+            modules.append((ident, load_configuration(test_file.name)))
+    for ident, module in modules:
+        assert ident == module.identifier
```

Comment:
Should we also test the increasing of the index for the name of the modules? Although I must admit I am not sure how best to do this, given that the result depends on whether `test_load_pyconfig_many` is run alone or `test_load_pyconfig` is run before it.

Reply:
I also considered testing the name, but decided against it since it seems like an implementation detail to me.

It would certainly be possible to test whether the number is a) larger than the load index when starting the test and b) larger than the previous module's index. However, it requires pattern matching the number from the module name.

Reply:
Thanks, I also only came up with ideas requiring pattern matching which seems like an unreasonable overhead for the gain from such a test.